cpp/src/unary/math_ops.cu

```diff
@@ -278,6 +293,12 @@ struct fixed_point_abs {
   __device__ T operator()(T data) { return numeric::detail::abs(data); }
 };
 
+template <typename T>
+struct fixed_point_negate {
+  T n;
```

Comment:
what's n used for?

Reply:
It looks like a throwaway, `fixed_point_abs` uses it too.

Reply:
Because we call `FixedPointUnaryOpFunctor{n}`